spark-doris-connector/src/main/java/org/apache/doris/spark/util/ListUtils.java

```diff
@@ -43,10 +43,11 @@ public static List<String> getSerializedList(List<Map<Object, Object>> batch) th
      * @throws JsonProcessingException
      */
     public static void divideAndSerialize(List<Map<Object, Object>> batch, List<String> result) throws JsonProcessingException {
-        String serializedResult = (new ObjectMapper()).writeValueAsString(batch);
+
         // if an error occurred in the batch call to getBytes ,average divide the batch
         try {
             //the "Requested array size exceeds VM limit" exception occurs when the collection is large
+            String serializedResult = (new ObjectMapper()).writeValueAsString(batch);
```

Comment:
Would it be better to create `new ObjectMapper()` only once?

Reply:
yes，thank you for your advice，I will impove it

Reply:
I have improved it by singleton pattern，would you like to take a look again? Thank you! @JNSimba